synapse/storage/data_stores/main/end_to_end_keys.py

```diff
@@ -330,6 +332,105 @@ def get_e2e_cross_signing_key(self, user_id, key_type, from_user_id=None):
             from_user_id,
         )
 
+    def _get_e2e_cross_signing_keys_bulk_txn(
+        self, txn, users, key_type, from_user_id=None
+    ):
+        """Returns the cross-signing keys for a set of users.
+
+        Args:
+            txn (twisted.enterprise.adbapi.Connection): db connection
```

Comment:
You can document types like:

```python
def _get_e2e_cross_signing_keys_bulk_txn(
    self, txn: Connection, users: Iterable[str], key_type: str, from_user_id: str=None
) -> Deferred:
```

(Where `Iterable` is imported from `typing).